Formula/nmh.rb

```diff
@@ -1,21 +1,22 @@
 class Nmh < Formula
   desc "The new version of the MH mail handler"
   homepage "https://www.nongnu.org/nmh/"
-  url "https://download.savannah.gnu.org/releases/nmh/nmh-1.7.tar.gz"
-  sha256 "cd05c7ca2cae524ae99f6ba673463a5cdeff62df93e85913aa9277ae8304ce44"
+  url "https://download.savannah.gnu.org/releases/nmh/nmh-1.7.1.tar.gz"
+  sha256 "f1fb94bbf7d95fcd43277c7cfda55633a047187f57afc6c1bb9321852bd07c11"
 
-  bottle do
```

Comment:
please leave `bottle` section, CI will update it.

Reply:
Ok, done!